crypto/ui/ui_openssl.c

```diff
@@ -439,6 +439,16 @@ static int open_console(UI *ui)
             is_a_tty = 0;
         else
 #  endif
+#  ifdef EPERM
+            /*
+             * Linux can return EPERM (Operation not permitted),
+	     * e.g. if a daemon executes openssl via fork()+execve()
```

Comment:
Indentation seems to have gone wrong here

Reply:
I have fixed indentation, thank you.